--- sqliteman-1.2.2-orig/CMakeLists.txt	2010-06-19 13:13:06.000000000 -0600
+++ sqliteman-1.2.2/CMakeLists.txt	2012-08-02 00:27:29.000000000 -0600
@@ -132,7 +132,7 @@
     FIND_PACKAGE(QScintilla REQUIRED)
     IF (QSCINTILLA_FOUND)
         MESSAGE("QScintilla2 Found OK")
-        ADD_DEFINITIONS("-DQT -DSCI_LEXER")
+        set(BINDING_NAME QScintilla)
     ELSE (QSCINTILLA_FOUND)
         MESSAGE(STATUS "QSci: No QScintilla2 libraries found")
         MESSAGE(STATUS "QSci: You can use -DWANT_INTERNAL_QSCINTILLA=1 to build included copy of it.")
